panoramapublic/resources/queries/panoramapublic/peptideSearch.sql

```diff
@@ -0,0 +1,16 @@
+PARAMETERS
+(
+    peptideSequence VARCHAR,
+    exactMatch BIT default false
+)
+
+SELECT created,
+       count(Sequence) as matches,
+       title,
+       organism,
+       citation,
+       pxid,
+       peptideSequence AS peptideSequence,
```

Comment:
Do you also need to propagate exactMatch through like this?

Reply:
Let me test, I think I didn't need that because in case of exact match search user would be searching the complete peptideSequence which is getting propagated.

Reply:
done